README.md

```diff
@@ -247,6 +247,8 @@ extension|string (`'png' 'jpeg' 'webp' 'svg'`)|`'png'`      |File extension
 
 If you get an error running `npm install` referring to `node-pre-gyp`, this is caused by an attempt to compile the [`canvas` dependency](https://github.com/Automattic/node-canvas#compiling). See Compiling instructions in the README. For example on MacOS you need to install dependencies: `brew install pkg-config cairo pango libpng jpeg giflib librsvg pixman`.
 
+Currently this repo will not build (`npm run build`) on Node v18, recommended version is v16.
```

Comment:
Can we add a link or something (either here or in this issue) related to why it won't work on Node 18?

Reply:
Good shout, added a link to https://stackoverflow.com/q/69692842/1375972 

There are some other answers there pointing to solutions to fix it on v18. I was going to open an issue to come back to that at some point but don't seem to be able to open issues in this repo. But we've got the link here + in the readme for now